### DISCUSSION TITLE
A line of code is giving me an error and I need help figuring out why

### DISCUSSION THREAD

**Discussion by Amaranth-101** [community/community#138036] [Programming Help] [answered]
### Body

![IMG_0645](https://github.com/user-attachments/assets/8aa0c567-dacc-487e-aeca-db59e9a0f82b)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by davevad93** [comment, marked as answer]
Hi @Amaranth-101 ,

The error is due to incorrect indentation, fix it by doing this:

```python
if len(name) > 1:
   if name == "space":
      name = " "
   elif name == "enter":
      name = "[ENTER]\n"
   elif name == "decimal":
      name = "."
   else: 
      name = name.replace(" ", "_")
      name = f"[{name.upper()}]"
self.log += name 
```

**Reply by Amaranth-101** [reply]
Thanks!

**Reply by davevad93** [reply]
You are welcome @Amaranth-101 , next time you have a programming related question here in the Community I advise you to just paste your code, which is better than attaching a picture also for your privacy.

To attach a nicely-formatted code block you can do the following, for example in `python`:

````text
```python
print('Hello, world!')
```
````

For other languages it's the same but instead of "python" you put the name of the language you want to use, for example `html`.

````text
```html
<!DOCTYPE html>
<html>
<body>

   <h1>Heading</h1>
   <p>Paragraph.</p>

</body>
</html>
```
````
